### PR TITLE
ft raw aot test

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: tests/integration/llm/fastertransformer-model.py [hunk 1]
```diff
@@ -7,11 +7,22 @@
 def load_model(properties):
     tensor_parallel_degree = properties["tensor_parallel_degree"]
     pipeline_parallel_degree = 1  # TODO: add tests for pp_degree > 1
-    model_id = properties["model_id"]
+    model_id = properties.get('model_id') or properties.get('model_dir')
     dtype = properties.get("dtype", "fp32")
     return fastertransformer.init_inference(model_id, tensor_parallel_degree,
                                             pipeline_parallel_degree, dtype)
 
+def partition(inputs:Input):
```
Comment (frankfliu) [inline tests/integration/llm/fastertransformer-model.py:15]: It's a bit odd to passing Input here
1. It's unnecessary to use Input 
2. make the required parameter explicitly can help reading
3. We can are use kwargs, which allows user to customize partition script

Comment (frankfliu) [reply tests/integration/llm/fastertransformer-model.py:15]: Please format python code:

```
gradlew fP
```

Comment (sindhuvahinis) [reply tests/integration/llm/fastertransformer-model.py]: formatting done.
